docs/reference/catalog-reference.md

```diff
@@ -1,6 +1,292 @@
+| `data_product_id` | **str / required**<br/>The data product ID.                                                         |
+| `image_id`        | **str / required**<br/>The scene ID.                                                                |
+| `aoi`             | **Union[dict, Feature, FeatureCollection, list, GeoDataFrame, Polygon]**<br/>The AOI to be ordered. |
+| `tags`            | **list[str]**<br/>A list of tags that categorize the order.                                         |
+
+<h5> Example </h5>
+
+```python
+catalog.construct_order_parameters(
+    data_product_id="647780db-5a06-4b61-b525-577a8b68bb54",  # Use catalog.get_data_products() to select a data product
+    image_id="a4c9e729-1b62-43be-82e4-4e02c31963dd",  # Use catalog.search() to select a scene
+    aoi="/Users/max.mustermann/Desktop/aoi.geojson",
+    tags=["project-7", "optical"],
+)
+```
+
+### estimate_order()
+
+The `estimate_order()` function returns a cost estimation for a catalog order.
+
+```python
+estimate_order(order_parameters)
+```
+
+The returned format is `int`.
+
+<h5> Arguments </h5>
+
+| Argument           | Overview                                                                      |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `order_parameters` | **Union[dict, None] / required**<br/>Parameters with which to place an order. |
+
+<h5> Example </h5>
+
+```python
+catalog.estimate_order(order_parameters) # Use catalog.construct_order_parameters() to get order_parameters
+```
+
+## Visualization
+
+### plot_coverage()
+
+The `plot_coverage()` function allows you to visualize the coverage of your search results.
+Use together with [`search()`](#search).
+
+```python
+plot_coverage(
+    scenes,
+    aoi,
+    legend_column,
+    figsize,
+)
+```
+
+<h5> Arguments </h5>
+
+| Argument        | Overview                                                                                                                                               |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `scenes`        | **GeoDataFrame / required**<br/>The catalog search results to be visualized.                                                                           |
+| `aoi`           | **GeoDataFrame**<br/>The AOI to be visualized.                                                                                                         |
+| `legend_column` | **str**<br/>The column name of `scenes` to arrange legend entries by ascending order. The default value is `sceneID`.                                  |
+| `figsize`       | **tuple[int, int]**<br/>The size of the visualization in inches. The first number is length, the second one is width. The default value is `(12, 16)`. |
```

Comment:
@dustgalactic 
FYI: The `matplotlib` library which is [used for all plotting](https://github.com/up42/up42-py/blob/7604b1ab1e6aed1990b264d3fc6e69a98811f6bd/up42/viztools.py#L20C10-L20C10) has their default measurement unit as inches.

> The native figure size unit in Matplotlib is inches, deriving from print industry standards.

https://matplotlib.org/stable/gallery/subplots_axes_and_figures/figure_size_units.html